lib/matplotlib/legend_handler.py

```diff
@@ -88,6 +88,58 @@ def adjust_drawing_area(self, legend, orig_handle,
         height = height - self._ypad * fontsize
         return xdescent, ydescent, width, height
 
+    def scale_dimensions(self, legend, width, height, orig_handle):
```

Comment:
Should this be private?

Reply:
scale dimensions is called by legend so it's public.

Reply:
We can call private methods from w/in the rest of Matplotlib.  I don't think there is any stricture that private means module-only.  I don't see any reason for this to be part of the public API (i.e. appear on the webpage).